html-aam/index.html

```diff
@@ -5124,7 +5124,16 @@ <h4 id="el-meter">`meter`</h4>
             <!-- <th><a href="#accessible-name-and-description-computation">Naming Algorithm</a></th> -->
             <tr>
               <th>Comments</th>
-              <td></td>
+              <td class="comments">
+                <div>
+                  If the `meter` element's subtree contains text content, user agents SHOULD expose this content, along with the element's initial value, by exposing the fallback content via the
```

Comment:
@cookiecrook's [migrated comment](https://github.com/w3c/html-aam/pull/401/files#r1382295857) provided below:

> element's subtree contains text content, user agents SHOULD expose this content

Should this use "text node"?
- https://github.com/w3c/accname/issues/212 

Or `innerText`?

> along with the element's initial value, by exposing the fallback content 

I dunno what this means... Does this mean concat the innerText and aria-valuenow, or convert aria-valuenow to a percentage and then concat? Does "along with" mean prepend? append? `join(", ")`?